Header header
# All velocities normalised to [-1,1]
float64 base_rotation_velocity
float64 arm_actuator_1_velocity
float64 arm_actuator_2_velocity
float64 wrist_rotation_velocity
float64 wrist_actuator_velocity
float64 gripper_velocity
